src/elliptic/curves/secp256_k1.rs

```diff
@@ -460,6 +460,15 @@ impl ECPoint<PK, SK> for Secp256k1Point {
     }
 }
 
+static mut CONTEXT: Option<Secp256k1<VerifyOnly>> = None;
+pub fn get_context() -> &'static Secp256k1<VerifyOnly> {
+    static INIT_CONTEXT: Once = Once::new();
+    INIT_CONTEXT.call_once(|| unsafe {
+        CONTEXT = Some(Secp256k1::verification_only());
+    });
+    unsafe { CONTEXT.as_ref().unwrap() }
+}
```

Comment:
Drive by comment: Did you consider using `lazy_static` for this?
e.g. https://github.com/comit-network/comit-rs/blob/master/vendor/secp256k1_support/src/lib.rs#L15

Reply:
Yes I have. that's basically what's lazy_static is doing and this doesn't require a new dependency